src/main/java/StringCalculatorStep8.java

```diff
@@ -0,0 +1,17 @@
+import java.util.Arrays;
+
+public class StringCalculatorStep8 {
+    public static final int Add(final String numbers) {
+        int returnValue = 0;
+        String[] numbersArray = numbers.split(",");
+        if (numbersArray.length > 2) {
```

Comment:
Accessing array elements directly with index can lead to IndexOutOfBoundsException. Handle this more gracefully.